### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'parent'

### ISSUE THREAD

**Issue by Sohtoren** [wikid24/ffxiv_mmd_tools_helper#11]
Hello,

I tried to follow the guide and after installing mek_tool, mmd_tools and ffxiv_mmd_tools_helper, when I try to read a chara file, I get this error : 

```
.chara file: E:\Users\Sohtoren\Anamnesis\Characters\CharName_Metadata.chara
Traceback (most recent call last):
  File "C:\Users\Sohtoren\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\ffxiv_mmd_tools_helper\import_ffxiv_charafile.py", line 790, in execute
    main(context,filepath,apply_charafile_to_selected=False)
  File "C:\Users\Sohtoren\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\ffxiv_mmd_tools_helper\import_ffxiv_charafile.py", line 695, in main
    selected_armature = model.findArmature(obj)
  File "C:\Users\Sohtoren\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\ffxiv_mmd_tools_helper\model.py", line 86, in findArmature
    if obj.parent.parent is not None:
AttributeError: 'NoneType' object has no attribute 'parent'
```

The chara has been exported with metadata. I tried with both Blender 4.0 and 3.6 LTS and I get the same error with both.

Here's the content of said chara file:

`{
  "FileExtension": ".chara",
  "TypeName": "Anamnesis Character File",
  "SaveMode": "All",
  "ModelType": 0,
  "ObjectKind": "Player",
  "Race": "Elezen",
  "Gender": "Feminine",
  "Age": "Normal",
  "Height": 100,
  "Tribe": "Wildwood",
  "Head": 1,
  "Hair": 112,
  "EnableHighlights": true,
  "Skintone": 1,
  "REyeColor": 88,
  "HairTone": 6,
  "Highlights": 0,
  "FacialFeatures": "Second, Fifth, Sixth, Seventh",
  "LimbalEyes": 5,
  "Eyebrows": 2,
  "LEyeColor": 88,
  "Eyes": 1,
  "Nose": 1,
  "Jaw": 3,
  "Mouth": 131,
  "LipsToneFurPattern": 183,
  "EarMuscleTailSize": 87,
  "TailEarsType": 3,
  "Bust": 100,
  "FacePaint": 3,
  "FacePaintColor": 63,
  "MainHand": {
    "Color": "0, 0, 0",
    "Scale": "0, 0, 0",
    "ModelSet": 601,
    "ModelBase": 2,
    "ModelVariant": 5,
    "DyeId": 0
  },
  "OffHand": {
    "Color": "0, 0, 0",
    "Scale": "0, 0, 0",
    "ModelSet": 698,
    "ModelBase": 1,
    "ModelVariant": 1,
    "DyeId": 0
  },
  "HeadGear": {
    "ModelBase": 9156,
    "ModelVariant": 1,
    "DyeId": 0
  },
  "Body": {
    "ModelBase": 1,
    "ModelVariant": 1,
    "DyeId": 102
  },
  "Hands": {
    "ModelBase": 239,
    "ModelVariant": 1,
    "DyeId": 6
  },
  "Legs": {
    "ModelBase": 6107,
    "ModelVariant": 1,
    "DyeId": 0
  },
  "Feet": {
    "ModelBase": 6002,
    "ModelVariant": 1,
    "DyeId": 0
  },
  "Ears": {
    "ModelBase": 142,
    "ModelVariant": 1,
    "DyeId": 0
  },
  "Neck": {
    "ModelBase": 1,
    "ModelVariant": 2,
    "DyeId": 0
  },
  "Wrists": {
    "ModelBase": 53,
    "ModelVariant": 1,
    "DyeId": 0
  },
  "LeftRing": {
    "ModelBase": 53,
    "ModelVariant": 1,
    "DyeId": 0
  },
  "RightRing": {
    "ModelBase": 53,
    "ModelVariant": 1,
    "DyeId": 0
  },
  "SkinColor": "0.8420761, 0.77854675, 0.71088046",
  "SkinGloss": "0.0625, 0.0625, 0.0625",
  "LeftEyeColor": "0.7647674, 0.46560556, 0.48179933",
  "RightEyeColor": "0.7647674, 0.46560556, 0.48179933",
  "LimbalRingColor": "0.08421377, 0.08421377, 0.08421377",
  "HairColor": "0.03254133, 0.03254133, 0.03254133",
  "HairGloss": "0.17937717, 0.17937717, 0.17937717",
  "HairHighlight": "0.93065745, 0.93065745, 0.93065745",
  "MouthColor": "0.016747408, 0.0007535563, 0.0034602077, 0.29411766",
  "BustScale": "1.08, 1.2, 1.184",
  "Transparency": 1,
  "MuscleTone": 1,
  "HeightMultiplier": 1.039,
  "Author": "Sohtoren"
}`

**Comment by diyachan** [comment]
for some reason, you need to load up a base before it works. not sure if you need the same race but should be like that or at least something close

**Comment by wikid24** [comment]
fixed in latest version 0.877
